README.md

```diff
@@ -58,7 +58,7 @@ For development work the following specifications have worked well in the past:
 * m5.xlarge
 * 80 GB disk
 
-In `dev` mode Docker containers will be built from source and the VM will run them during testing. You should make a note of your security group name and ID and ensure you can connect via ssh.
+In `dev` mode Docker containers will be built from source and the VM will run them during testing. You should make a note of your security group name and ID and ensure you can [connect via ssh](#makeip).
```

Comment:
You'll need to be able to ssh to the machine in both prod mode and dev mode. Notes on ssh should not be scoped to just dev mode.

Reply:
I rearranged several paragraphs (as discussed) and the new order makes much more sense to me. Also, the ssh cross link points to the point 7, which describes how to connect to the VM via ssh.